### PR TITLE
DOCS/COMMENTS: fixed typos

### PR DESCRIPTION
**Pull request by hoopoepg**

### CODE REVIEW

**Review by brminich — APPROVED** [review]

**Comment by hoopoepg** [comment]
bot:pipe:retest

**Comment by shamisp** [comment]
👍  @tonycurtis please take a look. thanks

File: src/ucp/api/ucp.h [hunk 1]
```diff
@@ -2959,7 +2959,7 @@ ucs_status_ptr_t ucp_am_send_nb(ucp_ep_h ep, uint16_t id,
  * @param [in]  param         Operation parameters, see @ref ucp_request_param_t.
  *
  * @note Sending only header without actual data is allowed and is recommended
- *       for transfering latency-critical amount of data.
+ *       for transferring latency-critical amount of data.
```
Comment (tonycurtis) [inline src/ucp/api/ucp.h:2962]: transferring a latency-critical...

File: src/ucp/api/ucp.h [hunk 2]
```diff
@@ -2735,7 +2735,7 @@ ucs_status_t ucp_mem_advise(ucp_context_h context, ucp_mem_h memh,
  *
  * @note
  * @li RKEYs for InfiniBand and Cray Aries networks typically includes
- * InifiniBand and Aries key.
+ * InfiniBand and Aries key.
```
Comment (tonycurtis) [inline src/ucp/api/ucp.h:2738]: Confusing.

File: src/uct/ib/dc/dc_mlx5_ep.c [hunk 3]
```diff
@@ -1215,7 +1215,7 @@ uct_dc_mlx5_iface_dci_do_common_pending_tx(uct_dc_mlx5_ep_t *ep,
         return UCS_ARBITER_CB_RESULT_STOP;
     }
 
-    /* No any other pending operations (except no-op, flush(CANEL), and others
+    /* No any other pending operations (except no-op, flush(CANCEL), and others
```
Comment (tonycurtis) [inline src/uct/ib/dc/dc_mlx5_ep.c:1218]: "No any"?

File: src/uct/ib/dc/dc_mlx5_ep.c [hunk 4]
```diff
@@ -1215,7 +1215,7 @@ uct_dc_mlx5_iface_dci_do_common_pending_tx(uct_dc_mlx5_ep_t *ep,
...
      * which don't consume TX resources) allowed to be still scheduled on an
```
Comment (tonycurtis) [inline src/uct/ib/dc/dc_mlx5_ep.c:1219]: which -> that

File: src/ucp/api/ucp.h [hunk 5]
```diff
@@ -2959,7 +2959,7 @@ ucs_status_ptr_t ucp_am_send_nb(ucp_ep_h ep, uint16_t id,
  * @param [in]  param         Operation parameters, see @ref ucp_request_param_t.
  *
  * @note Sending only header without actual data is allowed and is recommended
- *       for transfering latency-critical amount of data.
+ *       for transferring latency-critical amount of data.
  * @note The maximum allowed header size can be obtained by querying worker
  *       attributes by @ref ucp_worker_query routine.
```
Comment (tonycurtis) [inline src/ucp/api/ucp.h:2964]: by the @ref ...

**Review by tonycurtis — CHANGES_REQUESTED** [review]

File: src/ucp/api/ucp.h (hunk 1 above)
Comment (hoopoepg) [reply src/ucp/api/ucp.h]: updated

File: src/ucp/api/ucp.h (hunk 5 above)
Comment (hoopoepg) [reply src/ucp/api/ucp.h]: fixed

File: src/uct/ib/dc/dc_mlx5_ep.c (hunk 3 above)
Comment (hoopoepg) [reply src/uct/ib/dc/dc_mlx5_ep.c]: re-worded

File: src/uct/ib/dc/dc_mlx5_ep.c (hunk 4 above)
Comment (hoopoepg) [reply src/uct/ib/dc/dc_mlx5_ep.c]: updated

**Review by yosefe — APPROVED** [review]

**Comment by hoopoepg** [comment]
@yosefe ok to merge?

**Comment by brminich** [comment]
@tonycurtis, do you have other comments?

**Review by tonycurtis — APPROVED** [review]

**Pull request merged**